src/navigation.ts

```diff
@@ -239,10 +239,10 @@ export class Navigation {
     if (!e.workspaceId) {
       return;
     }
-    const flyoutWorkspace = Blockly.Workspace.getById(
-      e.workspaceId,
-    ) as Blockly.WorkspaceSvg;
-    const mainWorkspace = flyoutWorkspace.targetWorkspace;
+    const flyoutWorkspace = Blockly.Workspace.getById(e.workspaceId) as
```

Comment:
If you don't cast it, `getById` returns `WorkspaceSvg | null` and that would work just as well.

Reply:
It appears to be `Workspace | null`? But updated to `null` at least.